lib/portlayer/storage/vsphere/image.go

```diff
@@ -578,20 +578,20 @@ func (v *ImageStore) verifyImage(ctx context.Context, storeName, ID string) erro
 
 // XXX TODO This should be tied to an interface so we don't have to import exec
 // here (or wherever the cache lives).
-func inUse(ctx context.Context, ID string) error {
-	// XXX why doesnt this ever return an error?  Strange.
-	// Gather all the running containers and the images they are refering to.
-	state := exec.StateRunning
-	conts := exec.Containers.Containers(&state)
+func imagesInUse(ctx context.Context, ID string) error {
```

Comment:
Not necessarily.  The call to the container cache is a blocking one (or should be) and should therefore take a `ctx`.  Likewise, it should return an error (but isn't, ergo why this function returns an error).   I'll remove the `ctx` and the `error`.  The container cache could use some eyes TBH.